src/pages/subscription/00.subscription-page.tsx

```
@@ -0,0 +1,787 @@
+"use client";
+
+import { useState, useEffect } from "react";
+import { motion } from "framer-motion";
+import { Button } from "@/components/ui/button";
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "@/components/ui/card";
+import { Badge } from "@/components/ui/badge";
+import { Dialog, DialogContent, DialogDescription, DialogFooter, DialogHeader, DialogTitle } from "@/components/ui/dialog";
+import { useConvexQuery } from "@/lib/convex-query-hooks";
+import { api } from "../../../convex/_generated/api";
+import { Clock, Crown, Zap, ArrowRight, Calendar, CreditCard, History, Heart, Sparkles, CheckCircle2, Infinity, Star, Gift, Users, Gamepad2, BarChart3, TrendingUp } from "lucide-react";
```

Biome (2.1.2)

[error] Do not shadow the global "Infinity" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)